src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonInputFormatter.cs

```diff
@@ -105,6 +106,12 @@ public class JsonInputFormatter : TextInputFormatter
         }
 
         /// <summary>
+        /// Gets the flag indicating if it honors the value in setting <see cref="MvcOptions.ConvertAllExceptionsOfInbuiltInputFormattersToModelStateErrors"/>.
+        /// Default value is <see langword="true"/>.
+        /// </summary>
+        public virtual bool HandlesExceptions => GetType() == typeof(JsonInputFormatter);
```

Comment:
Should this be implicitly implemented? It's about the same amount of typing in derived types to override this anyway.

Reply:
We can't rely on dereived types to add code. That's a breaking change

Reply:
Not sure I follow. Wouldn't they have to override the property to opt in to this behavior?